### DISCUSSION TITLE
Fn + F hotkeys feature idea

### DISCUSSION THREAD

**Discussion by PoemOnTyperacer** [seerge/g-helper#599] [Ideas]
Hi, absolutely love your work!

Currently, when "Process Fn+F hotkeys without Fn" is enabled in Extras, the top row of keys performs the secondary functions (brightness/mute etc) whether you're holding Fn or not.
![image](https://github.com/seerge/g-helper/assets/63360610/b5c7608b-414a-4fae-a40c-f267d6ef98d3)
I think it could be useful to have "Process Fn+F hotkeys without Fn" invert the default situation, where a normal keypress calls the secondary function, but a Fn + key keypress calls the actual F1-12 key.

Thanks again for the awesome dev work!

**Comment by seerge** [comment]
@PoemOnTyperacer hello, that is not doable due technical reasons

**Comment by PoemOnTyperacer** [comment]
That's understandable, thanks for the quick answer!
